src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -156,50 +142,48 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(
         if (distributeToRecipients) {
             String optionFragment =
                     FeedbackQuestionFormTemplates.populateTemplate(optionFragmentTemplate,
-                            "${qnIdx}", Integer.toString(qnIdx),
-                            "${responseIdx}", Integer.toString(responseIdx),
-                            "${optionIdx}", "0",
-                            "${disabled}", sessionIsOpen ? "" : "disabled",
-                            "${constSumOptionVisibility}", "style=\"display:none\"",
-                            "${constSumOptionPoint}", existingConstSumResponse.getAnswerString(),
-                            "${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}", Const.ParamsNames.FEEDBACK_RESPONSE_TEXT,
-                            "${constSumOptionValue}", "");
+                            Const.FeedbackQuestion.QUESTION_INDEX, Integer.toString(qnIdx),
```

Comment:
TBH, I am not very convinced that extracting these strings into constants is an improvement. The string literals themselves appear in the 'template' html files that we have, so it will not be easy to change the values of these constants. @wkurniawan07 @damithc, any thoughts on this?